doc/radosgw/multisite.rst

```diff
@@ -604,9 +604,9 @@ Changing the Metadata Master Zone
 To promote a zone (for example, zone ``us-2`` in zonegroup ``us``) to metadata
 master, run the following commands on that zone::
 
-  $ radosgw-admin zone modify --rgw-zone=us-2 --master
```

Comment:
so, again this is all about consistency. i'd lean towards keeping as it is. and change it when we happen to touch this area.